src/components/CreateGroupForm/CreateGroupForm.tsx

```diff
@@ -115,11 +103,19 @@ export default function CreateGroupForm() {
   const autoComplete = useRef<HTMLInputElement>(null);
 
   useEffect(() => {
-    setClasses(query);
-    setFilteredClasses(query);
-  }, []);
+    if (isFormSubmitted) {
+      setTimeout(() => {
+        setIsFormSubmitted(false);
+      }, 1000);
+    }
```

Comment:
Can you explain this setTimeout? Why do you need to set FormSubmitted to false after 1 sec?

Reply:
I apologize. I used to use the timeout due to the asynchronous behavior of updating react states. However, now that it's within a useEffect, it's no longer required. Removing them now.